lib/demos/cupertino/cupertino_context_menu_demo.dart

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/gallery_localizations.dart';
+
+// BEGIN cupertinoContextMenuDemo
+
+class CupertinoContextMenuDemo extends StatelessWidget {
+  const CupertinoContextMenuDemo();
+
+  @override
+  Widget build(BuildContext context) {
+    final galleryLocalizations = GalleryLocalizations.of(context);
+    return CupertinoPageScaffold(
+      navigationBar: CupertinoNavigationBar(
+        automaticallyImplyLeading: false,
+        middle: Text(
+          galleryLocalizations.demoCupertinoContextMenuTitle,
+        ),
+      ),
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          Center(
+            child: Container(
+              width: 100,
+              height: 100,
+              child: CupertinoContextMenu(
+                child: Container(
+                  child: const FlutterLogo(size: 250),
```

Comment:
The parent container is 100x100 and this is 250, is that going to cause a problem?

Reply:
No it's not causing any problem , The size of flutter icon will be increased to 250 only when context menu will show up .